gssapi/tests/test_raw.py

```diff
@@ -1353,3 +1353,20 @@ def test_encode_from_int_seq(self):
             int_seq = oid['string'].split('.')
             o = gb.OID.from_int_seq(int_seq)
             o.__bytes__().should_be(oid['bytes'])
+
+    def test_comparisons(self):
+        krb5 = gb.OID.from_int_seq(TEST_OIDS['KRB5']['string'])
+        krb5_other = gb.OID.from_int_seq(TEST_OIDS['KRB5']['string'])
+        spnego = gb.OID.from_int_seq(TEST_OIDS['SPNEGO']['string'])
+
+        a = krb5 == krb5_other
```

Comment:
make this `(krb5 == krb5_other).should_be(True)`  so that we get nice error messages with should_be